implementation/src/main/java/io/smallrye/config/SmallRyeConfig.java

```diff
@@ -139,20 +139,44 @@
             }
         }
 
-        String value = configValue.getValue();
+        /*
```

Comment:
May be more interesting to move this to a java doc section.

Reply:
Is this what you meant? 
https://github.com/smallrye/smallrye-config/blob/39c034bf374d2de98ca066294a72722c185a0821/implementation/src/main/java/io/smallrye/config/SmallRyeConfig.java#L128-L142

Reply:
Yes.